cmd/snap-confine/ns-support.c

```diff
@@ -273,20 +273,47 @@ static dev_t find_base_snap_device(const char *base_snap_name,
 	return base_snap_dev;
 }
 
-static bool should_discard_current_ns(dev_t base_snap_dev)
+static bool homedirs_are_mounted(sc_mountinfo *mi, char **homedirs)
 {
```

Comment:
the diff of this file is quite messy, maybe there's a reording of the changes/additions to avoid this?

Reply:
Managed to clear this diff up